### PR TITLE
Better test coverage for Estonia personal ID codes

### PR DESCRIPTION
**Pull request by karlerikounapuu**
At the moment, some of PR tests are failing because coverage for Estonia personal ID codes is poor. Provides better test coverage.

### CODE REVIEW

File: lib/identity_code/estonia.rb [hunk 1]
```diff
@@ -37,7 +37,7 @@ def check_digit
 
       if checknum == 10
         scales = [3, 4, 5, 6, 7, 8, 9, 1, 2, 3]
-        checknum  = combine_scale_with_number(scales, identity_code)
+        checknum = combine_scale_with_number(scales, identity_code)
```
Comment (artur-beljajev) [inline lib/identity_code/estonia.rb]: Can we keep reformatting-related changes separately? Otherwise this change seems to have nothing to do with "Fix test coverage for Estonia personal ID codes".

File: test/lib/identity_code_test.rb [hunk 2]
```diff
@@ -30,4 +30,15 @@ def test_is_valid_randomly_for_estonian_id_codes
     identity_code = IdentityCode.new('EE', array_of_codes.sample)
     assert(identity_code.valid?)
   end
+
+  def test_is_valid_for_every_century
```
Comment (artur-beljajev) [inline test/lib/identity_code_test.rb]: - Looking at the name of the test, it's not clear what this test actually tests.
- I also find the tests themselves quite cryptic: why value `14501234213` is valid?
- Don't we need to test negative case (when a record is invalid)?

File: lib/identity_code/estonia.rb (hunk 1 above)
Comment (karlerikounapuu) [reply lib/identity_code/estonia.rb]: Alright. Opening a new PR just to fix one space feels hilarious to me, to be honest. I'll add the space back.

Comment (artur-beljajev) [reply lib/identity_code/estonia.rb]: I meant separate commit, actually :)

Comment (artur-beljajev) [reply lib/identity_code/estonia.rb]: Something that would explain the change, like "Reformat" (this is what I use, but feel free to choose whatever you think is clear for the widest audience)

Comment (artur-beljajev) [reply lib/identity_code/estonia.rb]: The point is that if you see such commit message, you know right away that no logic is changed there, and you don't spend time digging into the details.

File: test/lib/identity_code_test.rb (hunk 2 above)
Comment (karlerikounapuu) [reply test/lib/identity_code_test.rb]: - Agreed. This test can be named more accurately.
- For me it's self-explanatory, I assert that this ID code is valid. Why would one write a test that intentionally is not valid? 
- Yup, great idea.

**Comment by karlerikounapuu** [comment]
> > This PR pushes coverage to 100%.
> 
> Do you think this should be a goal?

Not a goal, but intention of this PR is to have better test coverage, yes. It was meant as a statement.

File: test/lib/identity_code_test.rb [hunk 3]
```diff
@@ -30,4 +30,30 @@ def test_is_valid_randomly_for_estonian_id_codes
...
+    identity_code = IdentityCode.new('EE', "38701234218")
+    assert(identity_code.valid?)
+
+    # Valid estonian ID code for person born in 21th century
+    identity_code = IdentityCode.new('EE', "60901234211")
+    assert(identity_code.valid?)
+
+    # Incorrect estonian ID codes
+    identity_code = IdentityCode.new('EE', "29511234213")
+    assert_not(identity_code.valid?)
```
Comment (artur-beljajev) [inline test/lib/identity_code_test.rb]: `assert identity_code.invalid?` ?

Comment (karlerikounapuu) [reply test/lib/identity_code_test.rb]: identity_code doesn't have function `invalid?`. I wouldn't create this method because it doesn't solve anything nor provide functionality that's _really_ needed.

Comment (artur-beljajev) [reply test/lib/identity_code_test.rb]: You're right! I assumed that it's `ActiveModel`-based for some reason... I would still create `invalid?` with the following simple content:

```
def invalid?
  !valid?
end
```

since this is what we're testing: that a record is invalid. Negations are less readable, so if it can be avoided, I would avoid it.

Comment (karlerikounapuu) [reply test/lib/identity_code_test.rb]: Okay, fine by me 😀 Adding your snippet and porting tests to use it as well.
